src/stdlib/Builtins.fs

```diff
@@ -10,13 +10,14 @@ open Fable.Core
 type TextIOBase =
     abstract read : unit -> string
     abstract read : __size: int -> string
+    abstract write : string -> unit
 
 type TextIOWrapper =
     inherit IDisposable
     inherit TextIOBase
 
 [<StringEnum>]
-type OpenTextModeUpdating =
+type OpenTextMode =
```

Comment:
So @dbrattli here I did something ugly. I tried to get the `OpenTextMode ` union working with its different open modes but I could only get it working by adding the read, write and update to the same union context. What would you recommend to do here?

Reply:
@MMagueta Yes this looks like the right thing to do 👍